### PR TITLE
Config SDK

### PR DESCRIPTION
**Pull request by devinmarieb**
# Description
Short overview page for the config SDK. The page links out to the official SDK pages.

# Reasons
We need a small page that references the SDK and makes it easier for developers to find.
[Jira ticket](https://circleci.atlassian.net/jira/software/c/projects/DOCTEAM/boards/554?modal=detail&selectedIssue=DOCTEAM-710)

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [x] Break up walls of text by adding paragraph breaks.
- [x] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [x] Keep the title between 20 and 70 characters.
- [x] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [x] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [x] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [x] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/circleci-config-sdk.adoc [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+:toc: macro
+:toc-title:
+
+[#overview]
+== Overview
+
+The link:https://circleci-public.github.io/circleci-config-sdk-ts[CircleCI Config SDK] is a JavaScript package library (also compatible with TypeScript) for generating a CircleCI YAML configuration file. Use the SDK to:
+
+* Replace writing YAML conguration files
+* Augment YAML configuration files
```
Comment (KyleTryon) [inline jekyll/_cci2/circleci-config-sdk.adoc:19]: I think this one might be true if it were included under `Enhance CircleCI's <<using-dynamic-configuration#,dynamic configuration>>`

I'm thinking for an alternative but I think we should remove this line

File: jekyll/_cci2/circleci-config-sdk.adoc [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+For example, generating a static configuration file can be useful for CLI tools where you want to create a CircleCI configuration, or browser-based tooling, such as a link:https://github.com/CircleCI-Public/visual-config-editor/[visual config editor].
+
+Components created with the CircleCI Config SDK can be packaged and distributed as Node packages.
+
+[#install-the-sdk]
+== Install the SDK
+
+Install the CircleCI Config SDK with NPM or Yarn, and include it in your project via Node.js or the browser.
+
+NPM installation:
```
Comment (kbatuigas) [inline jekyll/_cci2/circleci-config-sdk.adoc:33]: I don't think we have to implement it here (and we should probably have a larger discussion on it as well) but is this the kind of snippet where tabs would work well?

**Review by kbatuigas — APPROVED** [review]
Just one comment but no changes required!

File: jekyll/_cci2/circleci-config-sdk.adoc (hunk 2 above)
Comment (devinmarieb) [reply jekyll/_cci2/circleci-config-sdk.adoc:33]: Yeah, that's a good idea!

**Pull request merged**